go/vt/sqlparser/ast.go

```diff
@@ -916,6 +941,52 @@ type TriggerOrder struct {
 	OtherTriggerName  string
 }
 
+type ProcedureSpec struct {
+	Name            string
+	Definer         string
+	Params          []ProcedureParam
+	Characteristics []Characteristic
+	Body            Statement
+}
+
+type ProcedureParamDirection string
+const (
+	ProcedureParamDirection_In ProcedureParamDirection = "IN"
```

Comment:
All the other string constants defined in this file are lower case. Better to match that convention here.

Any time we match stuff in string-like fashion we tend to lower-case the user value and compare directly with the constant value. It doesn't actually apply here since these come out of the parser directly, but just better to be consistent.